### DISCUSSION TITLE
抽象高中生活（2）

### DISCUSSION THREAD

**Discussion by jsishsjsj** [share121/inter-knot#5197] [灌水]
同学抽象发言之没有腹肌就不能撩起来衣服散热……
是我不配了。

**Comment by Junjun280** [comment]
也是逆天的

**Comment by March-777th** [comment]
？<img src="https://bbs-static.miyoushe.com/static/2024/07/31/1a2616d760a6f334a4a9abca46267af3_7281603850910753959.png" alt="三月七 我" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/07/31/73346f2b98d45b62da9139d2774fa856_8641114165088315754.png" alt="三月七 告退" width="75" >
